src/coreservices.cpp

```diff
@@ -114,7 +115,7 @@ void CoreServices::initializeSettings() {
     m_pSettingsManager = std::make_unique<SettingsManager>(settingsPath);
 }
 
-void CoreServices::initialize(QApplication* pApp) {
+void CoreServices::initialize(QApplication* pApp, MixxxMainWindow* pMixxx) {
```

Comment:
Revert this!

Reply:
It is used for the window control in mpris.

Reply:
Coupling CoreServices to QWidgets is a big step backwards. Please find another way to implement this. Using signals and slots instead could work.

Reply:
That is not an issue, because it is only the forward declarated class. CoreServices passes only the pointer, without caring what it is.

Reply:
It is an issue to knowingly create problems for other branches in progress.

Reply:
Please explain the issue that it creates. 

We only pass a pointer. This can be even a void pointer in CoreService and we can cast it MixxxMainWindow in MPRIS. But making use of the compilers type safety is better.